app/grandchallenge/core/static/css/markdown.css

```diff
@@ -0,0 +1,31 @@
+@media (prefers-color-scheme: dark) {

```

Comment:
Maybe add a comment in this file to explain why it is necessary (bootstrap vs markdown css  conflicts.)